### PR TITLE
Show the launcher window in a menu bar item

### PR DESCRIPTION
**Pull request by Sidnicious**
@keybase/react-hackers

### CODE REVIEW

**Comment by cjb** [comment]
@Sidnicious I get a error trying to launch the tracker popup on Linux:

screenshot from 2015-11-12 14-50-36

File: desktop/app/dockIcon.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+var visibleCount = 0
+
+export default function () {
+  if (++visibleCount === 1) {
+    app.dock.show()
+  }
+  let alreadyHidden = false
+  return () => {
+    if (alreadyHidden) {
+      throw new Error('Tried to hide the dock icon twice')
```
Comment (chrisnojima) [inline desktop/app/dockIcon.js]: what will this actually do? kill the app?

Comment (Sidnicious) [inline desktop/app/dockIcon.js]: Hopefully, or at least make Electron throw an error. Could just make this a noop instead, but I figure this is better to catch our errors.

**Comment by Sidnicious** [comment]
@cjb Ah, sorry about that. I assumed the dock functions would noop on Linux, but this behavior makes more sense. Should be fixed!

**Comment by cjb** [comment]
@Sidnicious Nice, works great!  :thumbsup: to merge in.

**Comment by cjb** [comment]
Oh yeah, please also file JIRA issues for the bugs we saw.. the "active app" in OS X's top bar not switching over to Electron when main window is launched, and a gap in the menubar icons after we quit?

**Comment by Sidnicious** [comment]
Will do!

**Pull request merged**